packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -172,34 +147,13 @@ A rogue app could only intercept the authorization code, but it wouldn't have ac
 
 <div class="full">
```

Comment:
these new diagrams are giant when I view locally. I think three quarters for all of them would be better